sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md

```diff
@@ -30,6 +30,16 @@ dotnet add package Azure.ResourceManager.LoadTesting --prerelease
 
 To create an authenticated client and start interacting with Microsoft Azure resources, see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/dev/mgmt_quickstart.md).
 
+```C#
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+
+// Code omitted for brevity
+
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+```
+
```

Comment:
We already added the link for authentication, so I think we could remove this sample code.